tools/lsp/ui/components/selection-popup.slint

```diff
@@ -151,10 +151,11 @@ component PopupInner inherits Rectangle {
 
             Rectangle {
                 Image {
-                    source: Icons.search;
-                    colorize: Palette.foreground;
-                    width: EditorSizeSettings.default-icon-width;
-                    height: EditorSizeSettings.default-icon-width;
+                    source: Icons.filter;
+                    colorize: Palette.alternate-foreground;
+                    width: EditorSizeSettings.default-icon-width * 0.8;
+                    height: EditorSizeSettings.default-icon-width * 0.8;
+                    y: filter-edit.height / 2 - self.height / 2;
```

Comment:
Is this necessary? I thought I had the image centered already by putting it into the Rectangle. Maybe the Rectangle needs a `height: 100%;` to fill all the vertical space?

Reply:
it was necessary as I changed the size and it then did not center